### ISSUE TITLE
Namespace does not follow DotNet/Nuget conventions

### ISSUE THREAD

**Issue by David-Rushton** [David-Rushton/morello.markdown#6]
Right now our top level namespace is Markdown.  Which is a problem for several reasons:

- I don't own Markdown
- NuGet [recommendations](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#id-prefix-reservation-criteria) suggest avoiding common/generic terms
- Which is aligned with DotNet [recommendations](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces)  `<CompanyName>.<Product|Technology>.<Feature>`
- Once imported there is a greater chance of namespace clashes

**Comment by David-Rushton** [comment]
This is a good chance to fix #7 at the same time.

**Comment by David-Rushton** [comment]
## Proposed Solution

We have a project/company name!  Morello.  With a registered domain `morello.tools`.  I'm going to reorganise the project to include this.

### New Namespace 

`Morello` will become our top-level namespace.  It is not currently used on NuGet.  A search of UK Companies House and Google suggests the name isn't currently used in tech.

For anyone importing the library 

```cs
using Morello;

namespace Local.Namespace;

public class SomeClass
{
  public void SomeMethod()
  {
    MarkdownConsole.Write("**bold text**");
  }
}
```

**Comment by David-Rushton** [comment]
Addressed in #8
